webpack.config.test.js

```diff
@@ -0,0 +1,40 @@
+const webpack = require('webpack');
+const autoprefixer = require('autoprefixer');
+
+module.exports = {
+  // watch: false,
+  context: __dirname,
+  devtool: 'inline-source-map',
+  node: {
+    fs: 'empty'
+  },
+  module: {
+    loaders: [
+      {
+        test: /(\.jsx?)$/,
+        exclude: /node_modules/,
+        loader: 'react-hot!babel',
+        presets: ['es2015', 'react', 'stage-0']
```

Comment:
these can be better expressed via  [`.babelrc`](https://babeljs.io/docs/usage/babelrc/) file